torchgen/gen_functionalization_type.py

```diff
@@ -400,42 +564,19 @@ def emit_inplace_functionalization_body(
         for e in translate(unwrapped_args_ctx, dispatcher_sig.arguments(), method=False)
     ]
 
-    if functional_op is None:
-        # We can't functionalize this inplace op, since we don't know what the corresponding functional op is.
-        return_type = (
-            dispatcher.returns_type(f.func.returns).remove_const_ref().cpp_type()
-        )
-        warn_str = f"""Note: the functionalization pass encountered an operator ({str(f.func.name)}) that it could not \
```

Comment:
We get to fully kill this code now, since every mutable op is guaranteed to have a functional variant